### PR TITLE
add a secret pink mode via query param

### PR DESCRIPTION
**Pull request by ameowlia**
image

### CODE REVIEW

**Comment by tcdowney** [comment]
lgtm

**Review by tcdowney — APPROVED** [review]

File: build/script.js [hunk 1]
```diff
@@ -68,11 +68,21 @@ function initialize() {
...
 window.addEventListener('load', () => {
     initialize();
 });
 
+function secretPinkMode() {
+    var pink = new URLSearchParams(window.location.search).get('pink')
+    if (pink) {
+        document.getElementsByTagName("head")[0].insertAdjacentHTML(
+        "beforeend",
+        "<link rel=\"stylesheet\" href=\"pink.css\" />");
```
Comment (ericpromislow) [inline build/script.js:82]: I'd prefer outer quotes of `'` so the inner ones don't have to be backslashed

**Comment by ericpromislow** [comment]
> lgtm

:)

**Review by ericpromislow — APPROVED** [review]
I got "HOWDY" in 9, so it's not an impossible theme.

We'll deal with those quotes later

**Pull request merged**